sql/create-export-tracker-table.sql

```diff
@@ -0,0 +1,13 @@
+CREATE DATABASE sqs_entity_resolution
```

Comment:
Since this is only used by the docker entrypoint scripts, this should be in the `docker` directory.